storage_service/locations/models/gpg.py

```diff
@@ -293,16 +291,6 @@ def _encr_path2key_fingerprint(encr_path):
         raise GPGException(fail_msg)
 
 
-# This replaces non-unicode characters with a replacement character,
-# and is primarily used for arbitrary strings (e.g. filenames, paths)
-# that might not be valid unicode to begin with.
-# NOTE: non-DRY from archivematicaCommon/archivematicaFunctions.py
-def escape(string):
```

Comment:
I'm not sure the circumstances for this code previously, is it the use of `unicode_literals` that helps us to manage this when it's output in the new module?

Reply:
Exactly. Escape was only used on this line:

```
    detail = escape(
        "program=GPG; version={}; key={}".format(gpg_version, key_fingerprint)
    )
```
So previously it would have converted the bytestring to a unicode string, after the substitution of gpg_version and key_fingerprint. Now we're just using a unicode string directly. Both of the substitution vars should be already safe - version should be something like `1.4.16` and key_fingerprint comes from the database, so [Django should provide us with a unicode string](https://docs.djangoproject.com/en/1.8/ref/unicode/#models).